buildenv/jenkins/aqaTestPipeline.groovy

```diff
@@ -17,95 +17,126 @@ def TEST_FLAG = (params.TEST_FLAG) ?: ""
 
 def JOBS = [:]
 
-JDK_VERSIONS.each { JDK_VERSION ->
-    PLATFORMS.each { PLATFORM ->
-        String[] tokens = PLATFORM.split('_')
-        def os = tokens[1];
-        def arch = tokens[0];
-        if (arch.contains("x86-64")){
-            arch = "x64"
-        } else if (arch.contains("x86-32")) {
-            arch ="x32"
-        }
-
-        def filter = "*.tar.gz"
-        if (os.contains("windows")) {
-        	filter = "*.zip"
-        }
-        def short_name = "hs"
-        def jdk_impl = "hotspot"
-        if (params.VARIANT == "openj9") {
-            short_name = "j9"
-            jdk_impl = params.VARIANT
-        }
-
-        def download_url = params.CUSTOMIZED_SDK_URL ? params.CUSTOMIZED_SDK_URL : ""
-        def sdk_resource_value = SDK_RESOURCE
-
-        if (SDK_RESOURCE == "customized" ) {
-            if (params.TOP_LEVEL_SDK_URL) {
-                // example: <jenkins_url>/job/build-scripts/job/openjdk11-pipeline/123/artifact/target/linux/aarch64/openj9/*_aarch64_linux_*.tar.gz/*zip*/openj9.zip 
-                download_url = params.TOP_LEVEL_SDK_URL + "artifact/target/${os}/${arch}/${params.VARIANT}/${filter}/*zip*/${params.VARIANT}.zip"
+node ("master") {
```

Comment:
I think we cannot use `master`. If I remember it correctly, some Jenkins server does not have `master`.

Reply:
Yeah, forgot openj9 doesn't have master